### ISSUE TITLE
Открыта новая тема СВОДД 47

### ISSUE THREAD

**Issue by iprst** [terratensor/svodd#266]
ПРОДОЛЖЕНИЕ ОБСУЖДЕНИЯ СВОДД 47
[https://фкт-алтай.рф/qa/question/view-10592](https://xn----8sba0bbi0cdm.xn--p1ai/qa/question/view-10592)

Подключаю её к системе.
 
![image](https://github.com/terratensor/svodd/assets/129882753/7621b076-a6ab-4c28-bb5c-163fdc02ad69)

 
Статистика подключена, телеграм подключен.

![image](https://github.com/terratensor/svodd/assets/129882753/db838eec-be79-4d88-8d14-16871792e33c)

![image](https://github.com/terratensor/svodd/assets/129882753/fe8117c2-811a-46fb-9ea6-ee1566b9f765)

Системы работают штатно.
Полёт нормальный.

**Comment by iprst** [comment]
Что-то странное со счётчиком. 
Когда был добавлен второй комментарий в новую тему, счётчик показал 200%
После добавления в неё третьего комментария, счётчик показывает 150%

**Comment by audetv** [comment]
Сейчас уже отображается все верно, т.к. комментарии с ФКТ пришли. 
Скорее всего это продолжения бага при делении на ноль, когда от ФКТ было 0 комментариев, судя по коду я туда внес изменения и использовал выражение с тернарным оператором, зачем то, выглядит красиво, но не понятно, вот сейчас с ходу не могу прочесть, что это значит. Ни один раз уже замечал, что обычные `if {} else {}` читаются проще

```
 // Fixed FCT-SEARCH-CX [c114e2c19536b12101994a4eb1a9c56f][error][DivisionByZeroError]
    $total = ($svodd + $fct);
    $total = $total === 0 ? 1 : $total;
```
Так же более читаемо.
```
if ($total === 0) {
    $total = 1;
}
```
Пока дальше не понятно, я ранее наблюдал, что график себя ведет странно в начале, при смене темы, но не придавал значения, т.к. когда появляются комментарии с обоих сторон, все выглядит нормально. Вычитать с ходу в коде не получается, нужно углубиться. Посмотрю еще раз позже, новым взглядом так сказать.

**Comment by iprst** [comment]
Да, видимо это от деления на ноль, поскольку после появления комментариев в общих темах, график пришёл в нормальное состояние. В таком случае, возможно, имеет смысл заранее считать будущий комментарий с сайта за единицу и делить на один, а при появлении «первого» реального присваивать ему это значение.
